### PR TITLE
Fix logs output

### LINKED ISSUE #322: Aibolit have lot of logs
AT the moment Catboost  creates lots of unnecessary logs. They have to remove it every time the program is run

### PR DESCRIPTION
**Pull request by lyriccoder**
Fix #322

1) remove cat_boost folder with logs
2) fix \n in the logs. Now training has a good human-readable output

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#335](https://github.com/yegor256/aibolit/issues/335) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/CT5UB7XAP/37012cd8-cdf3-4941-84e5-30f908a30f1f); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @acheshkov[/z](https://www.0crat.com/u/acheshkov) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by acheshkov** [comment]
@0crat out

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/yegor256/aibolit/issues/335#issuecomment-633415677))

@acheshkov GitHub issue is still open, won't close

File: aibolit/ml_pipeline/ml_pipeline.py [hunk 1]
```diff
@@ -15,20 +15,24 @@ def collect_dataset(args):
...
             print(result.stdout)
-            dataset_file_path = Path(cur_work_dir, args.dataset_file)
-            print('Saving dataset to {}'.format(str(dataset_file_path.absolute())))
-            if not dataset_file_path.parent.exists():
-                dataset_file_path.parent.mkdir(parents=True)
-            shutil.copy(Path(Config.dataset_file()), dataset_file_path)
-
-    def run_cmd(metrics_cmd):
-        result = subprocess.run(metrics_cmd, stdout=subprocess.PIPE)
+            if args.dataset_file:
```
Comment (acheshkov) [inline aibolit/ml_pipeline/ml_pipeline.py:25]: please to create a separate function

**Comment by 0crat** [comment]
Job audit: performer @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)[/z](https://www.0crat.com/u/carlosmiranda) didn't make CR comments

**Pull request merged**